src/Compute/Compute.Test/ScenarioTests/DiskRPTests.cs

```diff
@@ -93,6 +93,11 @@ public void TestSnapshotConfigDiskAccessNetworkPolicy()
         {
             TestRunner.RunTestScript("Test-SnapshotConfigDiskAccessNetworkPolicy");
```

Comment:
The name of the PR needs to be more informative. Indicating the cmdlets at least.